### PR TITLE
Add entry for bhyve fixes for Intel and OpenBSD

### PR DESCRIPTION
**Pull request by anatoli26**

### CODE REVIEW

File: 2020q2/bhyve-intel-openbsd.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+processors with two VT-d translation units (IOMMU), included in most
+Skylake and newer Intel processors.
+
+The second bug was preventing correct interrupts handling for OpenBSD
+guests. As a result, OpenBSD guests running on bhyve were not able to
+use any PCI devices passed through to them from the host.
+
+During the last 2 months the second bug was identified and fixed and
+they both were backported to 12.1-RELEASE (p7). So now it's possible to
+fully take advantage of PCI passthrough with bhyve in a production-ready
```
Comment (debdrup) [inline 2020q2/bhyve-intel-openbsd.md:28]: Shortening PCI passthrough here is fine, but perhaps you should add a parenthesis mentioning the shortened form you use later?

File: 2020q2/bhyve-intel-openbsd.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+
+The most typical case for ppt is to pass to the guest network adapters
+for its complete control, but you can also pass through USB devices
+(including external HDDs). Note though, passthrough of VGA and GPU
+devices is not supported yet (for more details see the 3rd link).
+
+A particularly interesting case for ppt is to use OpenBSD guest as a
+firewall and a router for a FreeBSD server. OpenBSD is considered by
+many to be the most secure OS and it's frequently used in network
+appliances as a firewall/router.
```
Comment (debdrup) [inline 2020q2/bhyve-intel-openbsd.md:39]: Without meaning to cast aspersions on any other project, I don't believe it's appropriate for quarterly reports to serve as a direct promotional platform (although I realise that some work carries 'sponsored by' notices) - Can you please try rephrasing the sentence after the first period in this paragraph?

File: 2020q2/bhyve-intel-openbsd.md (hunk 1 above)
Comment (anatoli26) [reply 2020q2/bhyve-intel-openbsd.md:28]: Do you mean something like this: "So now it's possible to fully take advantage of PCI passthrough (ppt) with bhyve in a production-ready..."?

File: 2020q2/bhyve-intel-openbsd.md (hunk 2 above)
Comment (anatoli26) [reply 2020q2/bhyve-intel-openbsd.md:39]: Does this sound better: "OpenBSD has a solid track record in security and is frequently used in network appliances as a firewall/router."?

**Pull request merged**